src/main/resources/errors.yaml

```diff
@@ -458,8 +458,35 @@ request-errors:
     code: INVALID_INDEX_DEFINITION
     title: Provided index configuration is not valid.
     body: |-
-      Provided index configuration is not valid: ${reason}.
-  
+      Provided index configuration is not valid: ${reason}.V
+
+  - scope: SCHEMA
+    code: COLUMN_ALREADY_EXISTS
+    title: Column already exist in the table.
+    body: |-
+      Column `${column}` already exist in the table.
+
+  - scope: SCHEMA
+    code: COLUMN_NOT_FOUND
+    title: Column doesn't exist in the table.
+    body: |-
+      Column `${column}` doesn't exist in the table.
+
+  - scope: SCHEMA
+    code: COLUMN_CANNOT_BE_DROPPED
+    title: Column can't be dropped.
+    body: |-
+      Column can't be dropped: ${reason}.
```

Comment:
Should have `${column}` passed for templating

Reply:
Appended the column as part of the reason 
`"Primary key column `%s` cannot be dropped".formatted(column.asCql(true))`

Reply:
I know but why not follow the usual way with templating, same as other messages?

Reply:
I am trying to balance on number of error codes that api emit vs providing meaningful error message. If I have to avoid formatting text in the code then it will end up with many types of error code.

Reply:
Ok in that case that makes sense. I don't want to force creation of multiple one-off codes for related cases.